cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSink.java

```diff
@@ -18,6 +18,12 @@
 
 package com.telefonica.iot.cygnus.sinks;
 
+import com.amazonaws.services.dynamodbv2.xspec.S;
```

Comment:
Used by?

Reply:
No, there isn't any new dependency, not very sure why I added it, but it's never used.

Fixed in [bf42de0](https://github.com/telefonicaid/fiware-cygnus/pull/1818/commits/bf42de008c59fb41d740ec6b851aee434e50fb7b)